pyhf/utils.py

```diff
@@ -162,15 +162,15 @@ def hypotest(poi_test, data, pdf, init_pars=None, par_bounds=None, **kwargs):
 
             to protect against excluding signal models in which there is little sensitivity. In the case that :math:`\textrm{CL}_{s} \leq \alpha` the given signal model is excluded.
 
-            - :math:`\left[\textrm{CL}_{s+b}, \textrm{CL}_{b}\right]`: The signal + background and background only :math:`p`-values as defined in Equations (75) and (76) of `arXiv:1007.1727`_
+            - :math:`\left[\textrm{CL}_{s+b}, \textrm{CL}_{b}\right]`: The signal + background :math:`p`-value and 1 minus the background only :math:`p`-value as defined in Equations (75) and (76) of `arXiv:1007.1727`_
```

Comment:
@lukasheinrich @kratsg I noticed I had a typo in the documentation RE: the CL_b and the p-values. This makes me think though, is the naming choice `return_p_values` a good one if we return `p_{s+b}` (a p-value) and `1-p_b` (not a p-value)? I'm not sure that using `return_probabilities` is better though, as I don't think it gets across the same information. Thoughts?

Reply:
I mean we could just return p_b and not CLb

Reply:
> That doesn't make it a p-value though, right? The p-value is determined by what is your Null hypothesis, and so the direction of the integration shouldn't be the defining factor(?). Maybe I'm confused.

Direction of integration is somewhat important. Isn't the main reason for having the CLs method to handle the situations in when signal < background and signal > background correctly -- and the corresponding statistical fluctuations?

![foo](https://kratsg.github.io/thesis/img/figuresTables/fig_122.png)

"... background (blue) and signal+background (red) hypotheses for 10 observed events (black line). In this example, p(10 | b) = 0.018 and p(10 | s+b) = 0.125. The observed data is more likely under the s+b hypothesis than background-only."

Reply:
the reason these plots above may be confusing (which is why I would not recommend including them in the docs) is that in profile likelihood (unlike in the examples above) we would use different test statistics to test the mu=0 and mu=1 hypothesis.. there, the null hypothesis always follows a ~chi square (wilks theorem) and we always integrate to the right

Reply:
Hm. Thanks for this discussion. I think I need to write a learn/lecture notebook (mostly for myself) on this now.

Reply:
I think I'll benefit from that too :)